cherrypy/_cperror.py

```diff
@@ -123,6 +123,9 @@ class Root:
 import six
 from six.moves import urllib
 
+from jaraco.classes.properties import classproperty
```

Comment:
No, it's okay. It's good that you explained your way of structuring imports. I think makes sense, we should probably document this approach in contribution docs to make everyone follow the same code style.